src/main/java/com/almanza/inventorystore/controller/impl/ProductControllerImpl.java

```diff
@@ -1,81 +1,46 @@
 package com.almanza.inventorystore.controller.impl;
 
 import com.almanza.inventorystore.controller.ProductController;
-import com.almanza.inventorystore.controller.dto.ProductDTO;
-import com.almanza.inventorystore.controller.dto.ProductToAddDTO;
-import com.almanza.inventorystore.controller.dto.ProductToUpdateDTO;
-import java.util.ArrayList;
-import java.util.LinkedList;
+import com.almanza.inventorystore.models.ProductModel;
+import com.almanza.inventorystore.models.ProductToAddModel;
+import com.almanza.inventorystore.models.ProductToUpdateModel;
+import com.almanza.inventorystore.service.ProductService;
 import java.util.List;
-import java.util.Optional;
+import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 @RequestMapping(path = "/inventory")
+@AllArgsConstructor
 public class ProductControllerImpl implements ProductController {
 
+  private ProductService productService;
+
   @Override
-  public ResponseEntity<List<ProductDTO>> getProduct() {
-    return ResponseEntity.status(HttpStatus.OK).body(List.of(
-        new ProductDTO(1,"Nike","Blue"),
-        new ProductDTO(2,"Puma","Black"),
-        new ProductDTO(3,"Adidas","White")
-    ));
+  public ResponseEntity<List<ProductModel>> getProduct() {
+    return ResponseEntity.status(HttpStatus.OK).body(productService.getProductService());
```

Comment:
when you call the products from service, you will get models, then you must map them to DTOs